Loop/Extensions/NSUserDefaults+StatusExtension.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  NSUserDefaults+StatusExtension.swift
+//  Loop
+//
+//  Created by Bharat Mediratta on 11/27/16.
+//  Copyright © 2016 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+extension UserDefaults {
+    
+    private enum Key: String {
+        case StatusExtensionContext = "com.loopkit.Loop.StatusExtensionContext"
+    }
+
+    class var appGroup: UserDefaults? {
+        get {
+            return UserDefaults(suiteName: "group.com.loudnate.Loop")
```

Comment:
This still needs to reflect the correct group ID. Use the `NSBundle` APIs to access the Info.plist keys/values. Because you can't ask which bundle UserDefaults belongs to, each client will need to determine this using a bundle-specific class like a view controller.

```swift
Bundle(for: MyClass.Type).object(forInfoDictionaryKey: "AppropriateKey")
```
Except I'd recommend creating a typed runtime property as an extension on `Bundle` to keep the key in one place, e.g. `.appGroupSuiteName` or something. 

Of course that means this property now has to take an argument... sorry about that.